forgotten-sender/listing-02/main.go

```diff
@@ -0,0 +1,56 @@
+// Example program showing a goroutine leak. We launch a goroutine that sends
+// on a channel but sometimes there is no other goroutine available to receive.
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"runtime"
+	"time"
+)
+
+func main() {
+
+	// Seed the random number generator so we get different results.
+	// The seed 42 always results in a timeout and thus a leak.
+	// The seed 99 always results in success and thus no leak.
+	// In production we may seed using the current time like
+	// rand.Seed(time.Now().UnixNano())
+	rand.Seed(42)
```

Comment:
You can't do this. There is nothing guarenteed here. Across different versions and architectures this could change. So let's just force the leak as before if that is important.

Reply:
Remove all this code.